kinto2xml/amo.py

```diff
@@ -16,6 +16,9 @@ def prepare_amo_records(records, fields):
         for field in list(record.keys()):
             if field not in fields:
                 del record[field]
+            else:
+                if isinstance(record[field], six.string_types):
+                    record[field] = record[field].strip()
```

Comment:
stripping could also happen during output.

Especially because the values are not stripped in web admin : see https://github.com/mozilla-services/amo-blocklist-ui/issues/68